extensions/wikia/Search/classes/Indexer.php

```diff
@@ -56,6 +56,16 @@ class Indexer {
 		'WikiViews'
 	];
 
+    /**
+     * A list of article descendant namespaces which are not allowed to be indexed for searching:
+     * Article Comments
+     * Forum
+     * Message Wall
+     *
+     * @var array
+     */
+    public static $excludedNamespaces =  [1, 1201, 2001];
```

Comment:
This way we also exclude `Talk` namespace if article comments are not enabled on the wiki. Is this expected behavior?